backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/utils/DeviceUtil.kt

```diff
@@ -0,0 +1,28 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2023 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.utils
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.LocalConfiguration
+
+@Composable
+fun isTablet(): Boolean {
+    val configuration = LocalConfiguration.current
+    return configuration.screenWidthDp > 513
```

Comment:
Why 513? Can we create a private constant for this?

Reply:
Checking again I don't know if this is still the best approach.
I'm considering going with the same approach we have for the main app (Sky_IsTablet, defining attributes under diff res folders), but quite not sure if there are better solutions under composable.

any opinions here @bvitaliyg @marianeum

Reply:
I think for now I would keep it private - later on we might want to extend this to not just handle the tablet vs phone but also small vs large tablet case that is defined in figma and expose that to have one source of truth